### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by tal-sitton**

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -15,14 +26,47 @@
...
+        intakePlacer.configureDashboard();
+        intakeRoller.configureDashboard();
+        intakeToTransfer.configureDashboard();
+        transfer.configureDashboard();
+        rootNamespace = new RootNamespace("Robot Namespace");
+        DigitalInput digitalInput = new DigitalInput(4);
+        rootNamespace.putBoolean("digital input 4", digitalInput.get());
+        rootNamespace.putData("intake cargo", new IntakeCargo());
+        rootNamespace.putBoolean("transfer limit", transfer::getStrapEntranceSensor);
+        rootNamespace.putData("test intake", new ParallelCommandGroup(
```
Comment (omri00) [inline src/main/java/frc/robot/Robot.java:55]: Test code shouldn't be in dev

File: src/main/java/frc/robot/Subsystems/Transfer.java [hunk 2]
```diff
@@ -1,4 +1,61 @@
...
+import frc.robot.RobotMap;
+
+import java.util.function.Supplier;
+
+/**
+ * Transports cargo from the bottom to the top of the robot.
+ */
+public class Transfer extends MotoredGenericSubsystem {
+
+    public static final double SPEED = -0.3;
```
Comment (omri00) [inline src/main/java/frc/robot/Subsystems/Transfer.java:17]: Make supplier

**Review by omri00 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Subsystems/IntakePlacer.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import frc.robot.RobotMap;
+
+/**
+ * Controls the position of the {@code IntakeRoller}.
+ *
+ * @see MotoredGenericSubsystem
+ */
+public class IntakePlacer extends MotoredGenericSubsystem {
+
+    public static final double MAX_SPEED = 0.5;
```
Comment (omri00) [inline src/main/java/frc/robot/Subsystems/IntakePlacer.java:16]: True for all subsystems:
If this is a hard mechanical limit on the motor, like giving 0.6 for 755 motors this is ok. If however this is just the speed for the commands this should be in the dashboard so you can change it at will if you see it's moving too slow or fast

File: src/main/java/frc/robot/Subsystems/IntakeToTransfer.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    private final DigitalInput limit;
+
+    public static IntakeToTransfer getInstance() {
+        if (instance == null) {
+            instance = new IntakeToTransfer(new WPI_VictorSPX(RobotMap.CAN.INTAKE_TO_TRANSFER_VICTOR));
+        }
+        return instance;
+    }
+
+    private IntakeToTransfer(WPI_VictorSPX victor) {
```
Comment (omri00) [inline src/main/java/frc/robot/Subsystems/IntakeToTransfer.java:27]: Do you need this to be a victor? If they change it to talon why change this in 2 places? 
True for all subsystems

**Review by omri00 — CHANGES_REQUESTED** [review]

**Review by OfriRosenbaum — APPROVED** [review]

**Review by OfriRosenbaum — APPROVED** [review]

**Pull request merged**